### ISSUE TITLE
vpasolve only supports scalar equations

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#459]
Is there an easy fix?  Otherwise, at least document this in the help text.

```
>> syms x y
>> eqn1 = exp(x) == x + 2;
>> eqn2 = 2*y == x;
>> vpasolve([eqn1; eqn2], [x; y], [0; 0])
error: Python exception: TypeError: cannot create mpf from [0]
    occurred at line 20 of the Python code block
error: called from
    python_cmd at line 176 column 5
    vpasolve at line 86 column 5
```

With a row vector we get a different error message:

```
>> vpasolve([eqn1 eqn2], [x y], [0 0])
error: Python exception: TypeError: bad operand type for abs(): 'MutableDenseMatrix'
    occurred at line 20 of the Python code block
error: called from
    python_cmd at line 176 column 5
    vpasolve at line 86 column 5
>> 
```

**Comment by cbm755** [comment]
Ok, `vpasolve` uses `mpmath.findroots`.  It does support multidimensional!

So this should be fixable.

**Comment by cbm755** [comment]
I thiink this was fixed in 2.6.0, at least with recent enough sympy.
